src/Game/zFX.cpp

```diff
@@ -154,22 +156,22 @@ void zFXGoo_SceneReset()
 
 #endif
 
-#if 1
+#if 0
 
 // func_800934EC
 #pragma GLOBAL_ASM("asm/Game/zFX.s", "zFXGoo_SceneExit__Fv")
 
 #else
 
```

Comment:
If this is decompiled, remove the if/else directives and just have the decompiled function